goguen/formal-spec/utxo.tex

```diff
@@ -951,6 +950,8 @@ \subsection{Updating the UTxO State}
       \\
       \mathsf{adaID}~\notin \dom~{\fun{forge}~tx} \\
       \forall txout \in \txouts{txb}, ~ \fun{getValue}~txout  ~\geq ~ 0 \\~
+      \forall txout \in \txouts{txb}, ~ \fun{getCoin}~txout ~\geq \\
+      \fun{valueSize}~(\fun{getValue}~txout) * \fun{minUTxOValue} pp \\~
       \\
       \fun{txsize}~{tx}\leq\fun{maxTxSize}~\var{pp} \\
       \fun{txexunits}~{txb} \leq \fun{maxTxExUnits}~{pp}
```

Comment:
We should have a call about what to do about ExUnits, I think tomorrow. I am going to schedule it. I think (based on discussions in the formal and rule-based spec meetings), we are going back to per-script budgets, which requires quite a bit of rule changes. Eg. , the `SVAL` transition type is no longer needed. But we still need a max memory threshold in addition to the `Steps` and `Memory-Steps` budget.